src/share/streams_nuopc/dshr_strdata_mod.F90

```diff
@@ -71,13 +70,14 @@ module dshr_strdata_mod
      type(ESMF_RouteHandle)              :: routehandle                     ! stream n -> model mesh mapping
      character(CS), allocatable          :: fldlist_stream(:)               ! names of stream file fields
      character(CS), allocatable          :: fldlist_model(:)                ! names of stream model fields
-     type(ESMF_FieldBundle)              :: fldbun_stream_lb                ! stream n field bundle for lb of time period (stream grid)
-     type(ESMF_FieldBundle)              :: fldbun_stream_ub                ! stream n field bundle for ub of time period (stream grid)
-     type(ESMF_FieldBundle)              :: fldbun_model_lb                 ! stream n field bundle for lb of time period (model grid)
-     type(ESMF_FieldBundle)              :: fldbun_model_ub                 ! stream n field bundle for ub of time period (model grid)
-     type(ESMF_FieldBundle)              :: fldbun_model                    ! stream n field bundle for model time (model grid)
-     integer                             :: ustrm                           ! index of vector u in stream
-     integer                             :: vstrm                           ! index of vector v in stream
+     integer                             :: stream_lb
+     integer                             :: stream_ub
+     type(ESMF_Field)                    :: field_stream                    ! a field on the stream data domain
```

Comment:
Can you please clarify if the field has the stream file name or the model filename. This was a huge source of confusion in the past.